src/docs/product/performance/resources.md

```diff
@@ -0,0 +1,120 @@
+  const duration = msToSec(entry.duration);
+
+  const span = Sentry.startInactiveSpan({
+    startTimestamp: startTime,
+    op: entry.initiatorType
+      ? `resource.${entry.initiatorType}`
+      : "resource.other",
+    description: entry.name,
+    data: {
+      "http.response_transfer_size": entry.transferSize,
+      "http.response_content_length": entry.encodedBodySize,
+      "http.decoded_response_content_length": entry.decodedBodySize,
+      "resource.render_blocking_status": entry.renderBlockingStatus,
+    },
+  });
+
+  // override end timestamp to match performance entry
+  span.finish(startTime + duration);
+});
+```
+
+## Resources Page
+
+The **Resources** page gives you a quick overview of your application's resource performance for the selected project(s). You can use this page as a starting point to investigate potential problem resources and drill down to better understand how various resources are affecting your app's performance.
+
+Open the **Resources** page by clicking "Resources" in the sidebar, under "Performance". Alternatively, the **Performance** page also surfaces the highest impact queries in the [TBD] widget. From there, you can click "View All" to open the **Resources** page.
+
+At the top of the page, summary graphs for requests per minute (throughput) and average duration provide high-level insight into the performance of your resources. If you see an anomaly or want to investigate a time range further, you can click and drag to select a range directly in a graph to filter data for that time range.
+
+The resource table below shows a list of grouped resources, along with their type, their volume (requests per min), average duration, the total time your app spent loading that resource (time spent), and [average encoded size](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming/encodedBodySize) of the resource.
+
+Currently only JavaScript and CSS resources are supported.
+
+<!-- TODO: Add information about time spent. Extract time spent docs into separate page? -->
+<!-- This paragraph also exists in queries table -->
+<!-- By default, this table is sorted by most [time spent](#what-is-time-spent). This means that queries at the top are usually really slow, very frequent, or both. You can change this to sort by queries per minute to see the most frequently run queries, or by average duration to see the slowest queries. -->
+
+To view more details, click on a resource from the table to open its **Resource Summary** page.
+
+### Resource Parameterization
+
+To enable Sentry to group similar resources together, Sentry parameterizes resource URLs, removing potenially dynamic elements. This helps track the performance of a particular resource across different releases, even when they have dynamic segments (used for busting caches or CDNS).
+
+To see an example of resource URL from a group, hover over a URL in the **Resource Summary** page.
+
+## Resource Summary Page
```

Comment:
Should we add a part that explains the charts, or at least the resource size chart which is first seen here

Reply:
added